ETC.tsv

```diff
@@ -2629,15 +2629,15 @@ ETC_20150317_002628	My Page
 ETC_20150317_002629	Last Visit
 ETC_20150317_002630	Open board and logout
 ETC_20150317_002631	Coin cycle
-ETC_20150317_002632	No Coin you can give.
-ETC_20150317_002633	Please enter the number of Coins shown.
+ETC_20150317_002632	You do not have any Coin to give.
+ETC_20150317_002633	Enter the amount of Coin you want to gift.
 ETC_20150317_002634	{Auto_2} ~ {Auto_1} ea
-ETC_20150317_002635	I received {Count} Coins as a gift from {Giver}.
-ETC_20150317_002636	I gifted {Count} Coins to {Receiver}.
-ETC_20150317_002637	{Giver} Mr. Gift Coins is missing.
-ETC_20150317_002638	{Receiver} can no longer receive Coins for today.
+ETC_20150317_002635	You received {Count} Coin as a gift from {Giver}.
```

Comment:
Personally I think it should be Coin(s). The plural sounds better for more than one.